packages/@unimodules/core/ios/UMCore/UMDefines.h

```diff
@@ -66,6 +66,9 @@ if (var == nil) { return; }
 // Converts nil -> [NSNull null]
 #define UMNullIfNil(value) (value ?: [NSNull null])
 
+// Converts "" -> [NSNull null]
+#define UMNullIfEmpty(value) (value == (id)[NSNull null] || value.length == 0 ? [NSNull null] : value)
```

Comment:
remove this - a macro that takes NSNull or NSStrings seems overly imprecise - probably want to write the code that uses this differently